robot/rospackages/src/mux_selector/scripts/mux_select_server.py

```diff
@@ -140,9 +140,8 @@ def mux_select_server():
         print('Exiting... Perhaps you mean\'t \'local\'?')
         sys.exit(0)
     if 'local' in non_ros_args:
-        print("Running in local mode")
```

Comment:
why? is it because we already have (local mode) ?

Reply:
exactly